pkg/ebpf/write_capture.go

```diff
@@ -92,7 +92,14 @@ func (t *Tracee) processFileWrites(ctx context.Context) {
 					continue
 				}
 				// note: size of buffer will determine maximum extracted file size! (as writes from kernel are immediate)
-				filename = fmt.Sprintf("bin.%d", mprotectMeta.Ts)
+				if t.config.Output.RelativeTime {
```

Comment:
Do we use this config option anywhere else besides the event's timestamp field right now?
Anyway if this is userfacing it should probably be documented.

Reply:
I think that's exactly the idea. In order to correlate the event and the captured files by timestamp, they need to have the same value.
Agree that we should document this option, though not directly related to this PR

Reply:
Not at all against the change, but if the scope of this flag is changed for this, maybe it should change for other timestamps as well (for example timestamps in arguments?).
Whatever we choose the scope of the flag needs to be clearly defined.

Reply:
I think in that case the scope of the flag is still the same since the only reason to have the timestamp in the filename is to correlate it to the event that generated it. Not sure that's the case for timestamps in arguments.